lib/index.ts

```diff
@@ -120,3 +121,13 @@ async function extractTargetFrameworksFromProjectConfig(
     throw new Error(`Extracting target framework failed with error ${err.message}`);
   }
 }
+
+async function containsPackageReference(manifestFileContents: string) {
+
+  const manifestFile: any = await parseManifestFile(manifestFileContents);
```

Comment:
my only comment here is performance. we parse packages.config to decide which manifest file to pick, and then we'll parse it again. Not sure how to improve it though

Reply:
This was my main concern as well. I originally parsed it just as text to find the reference but Mila pointed out that xml can have comments that could include the PackageReference as a word.

Reply:
From some of my testing the parsing was pretty fast alone, let's revisit if it shows to be a problem